src/pages/coding-standards/code-demarcation.md

```diff
@@ -0,0 +1,474 @@
+---
+title: Code Demarcation Standard | Commerce PHP Extensions
+description: Review code demarcation standards for Adobe Commerce and Magento Open Source extension development.
+---
+
+# Code demarcation standrad
+
+Core developers must follow the code demarcation standard.
+
+This standard is recommended for third-party [extension](https://glossary.magento.com/extension) developers.
+
+Some parts of code might not comply with the standard, but we are working to gradually improve this.
+
+The standard was developed in the scope of our efforts to ensure the following:
+
+-  Decouple visual (CSS) layer from the functional (JavaScript) layer.
+-  Decouple functional (JavaScript) layer from the [markup](https://glossary.magento.com/markup) (HTML).
+-  Reinstate emphasis on using of [jQuery](https://glossary.magento.com/jquery) templates.
+-  Reinstate emphasis on decoupling HTML, [CSS](https://glossary.magento.com/css) and JS from [PHP](https://glossary.magento.com/php) classes.
+
+Use [RFC 2119](http://www.ietf.org/rfc/rfc2119.txt) to interpret the "MUST," "MUST NOT," "REQUIRED," "SHALL," "SHALL NOT," "SHOULD," "SHOULD NOT," "RECOMMENDED," "MAY," and "OPTIONAL" keywords.
+
+## Semantics
+
+### For attribute names and values you must use meaningful unabbreviated lowercase words comprised of Latin characters concatenated with a hyphen (`-`)
```

Comment:
The right nav is messed up because the headings are way too long. Recommend retitling the long H3s, moving some of the text into a paragraph.